frontend/www/js/omegaup/common/index.ts

```diff
@@ -2,9 +2,12 @@ import Homepage from '../components/homepage/Homepage.vue';
 import { OmegaUp } from '../omegaup';
 import { types } from '../api_types';
 import Vue from 'vue';
+import * as ui from '../ui';
```

Comment:
The changes in this file should be moved to a new entrypoint.
```suggestion
import * as ui from '../ui';
```

If you have doubts on how to create it, don't hesitate to ask in slack. It could be added in a new PR